.github/workflows/coverage.yml

```
@@ -0,0 +1,47 @@
+    workflows: ["Integration Tests", "Unit Tests"]
+    types: [completed]
+
+jobs:
+  finalize:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.head_sha == github.sha }}
+    # Needed for listWorkflowRunsForRepo and safe defaults for token scope
+    permissions:
+      actions: read
+      contents: read
+      workflows: write
```

actionlint (1.7.7)

unknown permission scope "workflows". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)